transiXNOR/plotting.py

```diff
@@ -125,56 +128,121 @@
+	plt.tick_params(axis='both', which='minor', length=5, labelsize=MINOR_LABEL_SIZE)
+	ax.xaxis.set_minor_locator(AutoMinorLocator())
+	# plt.xlim([-0.1,0.3])
+	plt.ylim([1e-3, 1e2])
+	plt.savefig(model_path + '/plots/current_vds_' + '%.2f' % (Vds) + '_vbg_-0.1.pdf',
 		bbox_inches='tight', transparent=True)
 	plt.clf()
-	# vbg = 0.2 and vtg from 0 to 0.2 <--> vbg = 0 and vtg from 0.2 to 0.4
-	plt.semilogy(vtg_array, cur[0,0,20:41], linewidth=2, color='k')
+	# vbg = 0.0 and vtg from -0.1 to 0.3 <--> vbg = 0 and vtg from -0.1 to 0.3
+	plt.semilogy(vtg_array, cur[0, 0, 10:51], linewidth=2, color='k')
 	plt.tick_params(axis='both', which='major', length=10, labelsize=MAJOR_LABEL_SIZE)
 	plt.tick_params(axis='both', which='minor', length=5, labelsize=MINOR_LABEL_SIZE)
 	ax.xaxis.set_minor_locator(AutoMinorLocator())
-	plt.ylim([1e-2, 1e2])
-	plt.savefig(model_path+'/plots/current_vds_0.2_vbg_0.2.pdf',
+	# plt.xlim([-0.1, 0.3])
+	plt.ylim([1e-3, 1e2])
+	plt.savefig(model_path + '/plots/current_vds_' + '%.2f' % (Vds) + '_vbg_0.0.pdf',
 		bbox_inches='tight', transparent=True)
-	plt.clf()	
+	plt.clf()
+	# vbg = 0.1 and vtg from -0.1 to 0.3 <--> vbg = 0 and vtg from 0.0 to 0.4
+	plt.semilogy(vtg_array, cur[0, 0, 20:61], linewidth=2, color='k')
+	plt.tick_params(axis='both', which='major', length=10, labelsize=MAJOR_LABEL_SIZE)
+	plt.tick_params(axis='both', which='minor', length=5, labelsize=MINOR_LABEL_SIZE)
+	ax.xaxis.set_minor_locator(AutoMinorLocator())
+	# plt.xlim([-0.1, 0.3])
+	plt.ylim([1e-3, 1e2])
+	plt.savefig(model_path + '/plots/current_vds_' + '%.2f' % (Vds) + '_vbg_0.1.pdf',
+				bbox_inches='tight', transparent=True)
+	plt.clf()
+	# vbg = 0.2 and vtg from -0.1 to 0.3 <--> vbg = 0 and vtg from 0.1 to 0.5
+	plt.semilogy(vtg_array, cur[0, 0, 30:71], linewidth=2, color='k')
+	plt.tick_params(axis='both', which='major', length=10, labelsize=MAJOR_LABEL_SIZE)
+	plt.tick_params(axis='both', which='minor', length=5, labelsize=MINOR_LABEL_SIZE)
+	ax.xaxis.set_minor_locator(AutoMinorLocator())
+	# plt.xlim([-0.1, 0.3])
+	plt.ylim([1e-3, 1e2])
+	plt.savefig(model_path + '/plots/current_vds_' + '%.2f' % (Vds) + '_vbg_0.2.pdf',
+		bbox_inches='tight', transparent=True)
+	plt.clf()
+	# vbg = 0.3 and vtg from -0.1 to 0.3 <--> vbg = 0 and vtg from 0.2 to 0.6
+	plt.semilogy(vtg_array, cur[0, 0, 40:81], linewidth=2, color='k')
+	plt.tick_params(axis='both', which='major', length=10, labelsize=MAJOR_LABEL_SIZE)
+	plt.tick_params(axis='both', which='minor', length=5, labelsize=MINOR_LABEL_SIZE)
+	ax.xaxis.set_minor_locator(AutoMinorLocator())
+	# plt.xlim([-0.1, 0.3])
+	plt.ylim([1e-3, 1e3])
+	plt.savefig(model_path + '/plots/current_vds_' + '%.2f' % (Vds) + '_vbg_0.3.pdf',
+				bbox_inches='tight', transparent=True)
+	plt.clf()
 
 if (COMBINE_CURRENT_VIA_SYMMETRY):
-	vdsmin=0.01; vdsmax=0.2; vdsN=20;
-	vbgmin=0.0; vbgmax=0.2; vbgN=21;
-	vtgmin=0.0; vtgmax=0.2; vtgN=21;
+	vdsmin=-10; vdsmax=30; vdsN=41;
```

Comment:
If possible, it would be better to use the real voltages instead of the scaled ones. 
The correct way to convert vds to scaled integer can be found in the "merge conflict" (which I just committed this morning). You need to pull once before pushing.